frontend/components/Visibility.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
+
+const Visibility = () => {
```

Comment:
did we decide with the designers if this will still be a feature? i can't remember lol

Reply:
I think we are continuing with this as a feature.